src/domain/authentication/__tests__/idleProvider.test.jsx

```diff
@@ -43,9 +43,7 @@ test.each([
 
     vi.advanceTimersByTime(advancedTimeMs);
 
-    React.act(() => {
-      fireEvent.focus(document);
-    });
+    fireEvent.focus(document);
```

Comment:
The React.act was needed here, because otherwise the tests warned about act-issues. Also, at least earlier, the fireEvent was not wrapped with act by default -- the UserEvents were, but we are not using them here.